FrontEnd/src/components/HeaderFooter/footer/top/FooterNavigation.jsx

```diff
@@ -1,80 +1,100 @@
 import css from "./FooterNavigation.module.css";
+import { HashLink } from "react-router-hash-link";
 
 const PAGE_NAVIGATION_LINKS = [
-    {
-        id: "pnl0",
-        title: "Головна",
-        link: "#"
-    },
-    {
-        id: "pnl1",
-        title: "Компанії",
-        link: "#"
-    },
-    {
-        id: "pnl2",
-        title: "Стартапи",
-        link: "#"
-    },
-    {
-        id: "pnl3",
-        title: "Про нас",
-        link: "#"
-    }
-]
+  {
+    id: "pnl0",
+    title: "Головна",
+    link: "/",
+  },
+  {
+    id: "pnl1",
+    title: "Компанії",
+    link: "#",
+  },
+  {
+    id: "pnl2",
+    title: "Стартапи",
+    link: "#",
+  },
+  {
+    id: "pnl3",
+    title: "Про нас",
+    link: "/#about-us",
```

Comment:
no need  to add `/` before anchor

Reply:
This is the anchor for the main page section and without slash it tries to find the anchor on the other page. So this slash is necessary since we need to have this functionality working from any place.